pkg/memtier/tracker_idlepage.go

```diff
@@ -378,13 +377,17 @@ func (t *TrackerIdlePage) countPages() {
 		scanEndTime := time.Now().UnixNano()
 		stats.Store(StatsPageScan{
 			pid:      pid,
-			scanned:  totScanned,
+			scanned:  *totScanned,
 			accessed: totAccessed,
 			written:  0,
 			timeUs:   (scanEndTime - scanStartTime) / int64(time.Microsecond),
 		})
 		scanStartTime = scanEndTime
 	}
+	// pageHandler is called for all matching pages in the pagemap.
+	// It counts number of pages accessed and written in a region.
```

Comment:
This comment appears now in two places (see line 287). I think it should be removed from here.

Btw, any "access" detected by the idlepage tracker can be a "read" or a "write". The tracker cannot distinguish between those. Therefore this comment sounds a bit misleading, as it mentions "accessed and written". While at it, would you remove mentioning the "write" from the comment (line 288), please?